src/test/java/com/epam/atm/homework5/pf/LogInPage.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.atm.homework5.pf;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LogInPage extends AbstractPagePF {
+    private static final String ID_LOG_IN_FIELD = "identifierId";
+    private static final String CSS_LOGIN_NEXT_BTN = "#identifierNext";
```

Comment:
no needed separated variables, use in @FindBy directly

Please fix here and for all Pages.